src/domain/Game.ts

```diff
@@ -39,4 +42,85 @@ export class Game {
     getWordBeingGuessed() : string {
         return this.wordBeingGuessed;
     }
+
+    isGameOver() : string {
```

Comment:
isGameOver grąžintų true arba false. Panašu, kad tu čia ieškai game state.
Pasidaryk, kad šitas metodas būtų 
getGameState() : 'won' | 'lost' | 'in-progress' {
   ...
}

Reply:
kai grąžini stringus, kad kažką nusakyt tai stenkis, kad jie būtų uniform case ir vietoj tarpų naudok - arba _